build.sbt

```diff
@@ -43,7 +44,16 @@ lazy val circeParser        = "io.circe"                %% "circe-parser"
 lazy val circeGenericExtras = "io.circe"                %% "circe-generic-extras" % circeVersion
 lazy val commonsTest        = "ch.epfl.bluebrain.nexus" %% "commons-test"         % commonsVersion
 lazy val journal            = "io.verizon.journal"      %% "core"                 % journalVersion
-lazy val scalaTest          = "org.scalatest"           %% "scalatest"            % scalaTestVersion
+
+lazy val kamonCore       = "io.kamon" %% "kamon-core"            % kamonVersion
```

Comment:
need to set independent versions for kamon modules as they are released independently.